docs/ABOUT.md

```diff
@@ -1,41 +1,18 @@
-
-Haskell is a purely functional, lazy, statically-typed programming
-language with type inference.
+Haskell is a purely functional, lazy, statically-typed programming language with type inference. 
+It has a beautiful and minimal syntax.
```

Comment:
I am copying this old comment so that it remains visible on the changed line. It is unchanged from before, so no additional response is necessary! It is only a reminder!

(I have not myself given additional thought to this issue, but if nobody else has any ideas I'm going to suggest the sentence move back to where it was before so it doesn't come in between the list of adjectives and the definitions of each adjective)

---



Previously, immediately after the list of traits describing Haskell (purely functional, lazy, etc.), there comes a description of each trait.

Now, there is this sentence in between. Does this interposition interfere with the reading and understanding of this text?

Alternative: Leaving this sentence about beautiful and minimal syntax where it originally was. Is that better or worse, without having a code example?

(I have not decided what my own answer to these two questions is yet, so I'll leave them for others to think about for now)

Reply:
Any thoughts from the other @exercism/haskell @exercism/haskell-contributors members?

Reply:
Looks like none of them are interested in responding yet. In that case, I will give my advice, with appropriate time for anyone to respond to this advice. Stand by.